lib/fathead/flask/parse.py

```diff
@@ -0,0 +1,178 @@
+                section['abstract'] = '{}<br><pre><code>{}</code></pre>'.format(section['paragraph'], section['example'])
+                self.sections.append(section)
+            elif section['paragraph'] and not section['example']:
+                section['abstract'] = '{}'.format(section['paragraph'])
+                self.sections.append(section)
+
+        self.parsed_data = self.sections
+
+    def get_data(self):
+        return self.parsed_data
+
+    def get_example(self, soup):
+        text_html = soup.findNext('pre')
+        if not text_html is None:
+            text = text_html.get_text()
+            text = text.strip('\n')
+            return text.replace('\n', '\\n')
+        else:
+            return ""
+
+    def get_url(self, soup):
+
+        link_html = soup.find('a', {'class':'headerlink'}, href=True)
+        if not link_html is None:
+            anchor=link_html["href"]
+        else:
+            anchor=''
+
+        if anchor:
+            parsed_url = '{}{}'.format(os.path.join(URL_ROOT, self.file_name), anchor)
+        else:
+            parsed_url = os.path.join(URL_ROOT, self.file_name)
+
+        return parsed_url
+
+    def parse_section(self, soup):
+        first_paragraph_html = soup.findNext('p')
+        if not first_paragraph_html is None:
+            first_paragraph = self.clean_formatting(first_paragraph_html.get_text())
+        else:
+            first_paragraph=''
+        title = self.clean_formatting(soup.get_text())
+        anchor = self.get_url(soup)
+        url = os.path.join(URL_ROOT, self.file_name)
+
+        return {
+            'title': title,
+            'paragraph': first_paragraph,
+            'anchor': anchor,
+            'url': url
+        }
+
+    def clean_formatting(self, text):
+        
+        """
+        Flask documentation contains '¶' symbol to represent hyperlinks
+        which needs to be removed. Double spaces needs to be 
+        converted to single spaces 
+        """
+        
+        text = text.replace('  ', ' ').replace('\n', ' ').replace('\\n', r'\\n').replace("Â¶","").strip()
+        return text
+
+
+class Writer(object):
+    
+    """
+    Writes the parsed data to 
+    output.txt file in the desired format
+    """
+    
+    def __init__(self, data):
+        self.data = data
+
+    def get_data(self, entry_type, title, redirect_data='', url='', abstract='', anchor=''):
+        return [
+            title,           # title
+            entry_type,      # type is article
+            redirect_data,   # no redirect data
+            '',              # ignore
+            '',              # no categories
+            '',              # ignore
+            '',              # no related topics
+            '',              # ignore
+            url,             # add an external link back to Dive Into Python
+            '',              # no disambiguation
+            '',              # images
+            abstract,        # abstract
```

Comment:
@shanmbic please ensure the `abstract` is wrapped in `<section class="prog__container>...</section>`

More information on Fathead template elements is [available here](https://docs.duckduckhack.com/resources/fathead-overview.html#formatting-the-abstract)